Sources/Sentry/SentryPerformanceTracker.m

```diff
@@ -51,7 +51,9 @@ - (SentrySpanId *)startSpanWithName:(NSString *)name operation:(NSString *)opera
         newSpan = [activeSpan startChildWithOperation:operation description:name];
     } else {
         SentryTransactionContext *context =
-            [[SentryTransactionContext alloc] initWithName:name operation:operation];
+            [[SentryTransactionContext alloc] initWithName:name
+                                                nameSource:kSentryTransactionNameSourceComponent
```

Comment:
I'm actually not sure about this nameSource being correct. After all it's users that can call this `startSpanWithName` method, and then the source should be `.custom`. So I think we might need to change this.

Reply:
I removed this change, so now we're only sending a name source in SentryUIEventTracker.